API.md

```diff
@@ -0,0 +1,58 @@
+# SegM8Pi API
+
+## class SegM8
+
+Create an object of type `SegM8` to communicate with a chain of a particular [SegM8 7-segment
+indicator modules](https://my.amperka.com/modules/SegM8).
+
+### `SegM8(pin_CE: int, device_count: int = 1)`
+
+Construct a new SegM8 object that uses the default hardware SPI bus.
+
+- `pin_CE`: the chip enable (also known as chip select or slave select) pin used to control the
+  shift-register latch. It can take the values **0** or **1**, which corresponds to the **CE0** or
+  **CE1** pins on the Raspberry Pi board.
```

Comment:
I think you must say that it says no only about CS. This assumes that all SPI1 or SPI2 pins are used.

Reply:
By default, the Raspberry Pi is equipped with one SPI bus, which has 2 default chip choices:
https://www.raspberrypi.org/documentation/hardware/raspberrypi/spi/
To use **SPI1**, you need to configure the file config.txt, which can be difficult for the average user.